tsconfig.json

```
@@ -21,6 +21,7 @@
     "noUnusedParameters": true,
     "incremental": true,
     "noFallthroughCasesInSwitch": true,
+    "noImplicitAny": false,
```

Is this needed? Just conscious we don't want _too_ many `any` if we can avoid it.

---

I can't remember why I added this - possibly because of the 1000000 TS errors? Let me see what happens when I remove it. I agree, we don't really want this, so maybe I need to suppress it on a file by file basis rather than across the entire repo.

---

Here's what happens when I remove it: #13384

---

Had a go at resolving some of them. The rest can be fixed too, but TS isn't great at times for these `services[service][variant]` style of object accessing. Makes the code quite complex looking just to get type safety.

Happy to keep this in for now and I can try fixing the remaining TS errors.

---

Yeah I think that's why I opted for the `noImplicityAny` in the end - to give me type safety so that I didn't suppress the errors & mask the problem if a new service was not added correctly. 

Good luck fixing the remaining errors ☘️ !